src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -20,30 +20,18 @@
     validate_registry_name
 )
 
-register_cli_argument('acr', 'registry_name',
-                      options_list=('--name', '-n'),
-                      help='The name of the container registry',
-                      completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
-register_cli_argument('acr', 'storage_account_name',
-                      help='The name of an existing storage account',
-                      completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
+register_cli_argument('acr', 'registry_name', options_list=('--name', '-n'), help='The name of the container registry', completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
+register_cli_argument('acr', 'storage_account_name', help='The name of an existing storage account', completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
```

Comment:
Or add "#pylint: disable=line-too-long" to disable line-too-long checks in this file.